horovod/ray/runner.py

```diff
@@ -8,6 +8,8 @@
 from typing import Dict, Callable, Any, Optional, List
 import logging
 
+from ray.util import remove_placement_group
```

Comment:
> WARNING: autodoc: failed to import module 'ray' from module 'horovod'; the following exception was raised:
No module named 'ray.util'

this line seems to break doc build; could you instead remove and just access it inline?

Reply:
yep, done